package.json

```diff
@@ -188,7 +188,8 @@
     "react-web-config": "^1.0.0",
     "react-webcam": "^7.1.1",
     "react-window": "^1.8.9",
-    "semver": "^7.5.2"
+    "semver": "^7.5.2",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz"
```

Comment:
Why not install it from npm?

Reply:
That's good point, there is 0.18.5 version on the npm and it is no longer updated there instead of this authors encourage to install it in this way [here](https://docs.sheetjs.com/docs/getting-started/installation/frameworks). But technically we can use it from npm, because 0.18.5 version got everything we need at the moment, so what's your thoughts?
cc @rlinoz

Reply:
Using CDNs like this one can be risky as the content can be altered at any time, making it difficult to guarantee the code's trustworthiness. On the other hand, we have a lot more assurance with npm. We know that when we install version 0.18.5, the code is going to stay the same. 

The final decision, of course, is up to Expensify. They need to determine if they're comfortable installing a package from this source cc @rlinoz

Reply:
Let's go with npm and if at any point in time we need to update the library we can bring it in Slack and maybe host the version we want ourselves?